const.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
```

Comment:
`from enum import Enum, auto`

Reply:
Separate lines is more legible (source: personal opinion🙂) and [recommended by PEP 8](https://www.python.org/dev/peps/pep-0008/#imports), though this case is mentioned as an acceptable exception.

Reply:
What I've seen everywhere (and what is implemented in e.g. [isort](https://github.com/PyCQA/isort)) is
one import per line for different libraries
for "from X import" keep the same line if 3 or less, if more than three then wrap as a set and have one per line but indented.